git.mk

```diff
@@ -372,6 +374,8 @@ $(srcdir)/.gitignore: Makefile.am $(top_srcdir)/git.mk $(top_srcdir)/configure.a
 			"*.gcno" \
 			"*.$(OBJEXT)" \
 			$(DEPDIR) \
+			"*.pem" \
+			"*_state[1|2]" \
```

Comment:
These should be added to CLEANFILES or moved to maybe better moved to the tempdir.